examples_v20x/exti_blink/exti_blink.c

```diff
@@ -0,0 +1,60 @@
+#include "ch32fun.h"
+
+// specify interrupt line
+void EXTI1_IRQHandler( void ) __attribute__((interrupt));
+
+// set state of the LED on PA10
+int state = 1;
+
+int main()
+{
+	SystemInit();
+
+	funGpioInitAll();
+
+	// configure PA10 for output and set it high
+	funPinMode( PA10, GPIO_CFGLR_OUT_10Mhz_PP );
+	funDigitalWrite( PA10, FUN_HIGH );
+
+    // configure PA 1 for floating input
```

Comment:
Please use tabs consistently.

Reply:
Sorry about that. I think I fixed it but I'll check it in GitHub. My setting is vscode must compensate for it somehow because it looks normal there, but not in GitHub.